components/skeleton/skeleton-group-mixin.js

```diff
@@ -0,0 +1,48 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { SkeletonMixin } from './skeleton-mixin.js';
+import { SubscriberRegistryController } from '../../controllers/subscriber/subscriberControllers.js';
+
+export const SkeletonGroupMixin = dedupeMixin(superclass => class extends SkeletonMixin(superclass) {
+
+	static get properties() {
+		return {
+			_anySubscribersInSkeleton: { state: true },
+		};
+	}
+
+	constructor() {
+		super();
+		this._anySubscribersInSkeleton = false;
+		this._skeletonSubscribers = new SubscriberRegistryController(this, 'skeleton', {
+			onSubscribe: this.onSubscriberChange.bind(this),
+			onUnsubscribe: this.onSubscriberChange.bind(this),
+			updateSubscribers: this._checkSubscribersSkeletonState.bind(this),
+		});
+	}
+
+	updated(changedProperties) {
+		if (changedProperties.has('skeleton')) {
+			this._skeletonSubscribers.updateSubscribers();
+		}
+	}
+
+	onSubscriberChange() {
+		this._skeletonSubscribers.updateSubscribers();
+	}
+
+	_checkSubscribersSkeletonState(subscribers) {
+		this._anySubscribersInSkeleton = [...subscribers.values()].some(subscriber => (
+			subscriber._skeleton || subscriber._anySubscribersInSkeleton
+		));
+
+		this.setSkeletonActive(this._skeleton || this._anySubscribersInSkeleton || this._setByParent);
+
+		subscribers.forEach(subscriber => {
+			subscriber.setSkeletonActive(this._skeletonActive);
+			subscriber.setSetByParent(this._skeletonActive && !subscriber._skeleton);
+		});
+
+		this._parentSkeleton?.registry?.onSubscriberChange();
+	}
```

Comment:
Not sure I'm suggesting actually changing all the names, but is this accurate?
```suggestion
	_checkSubscribersSkeletonState(subscribers) {
		this._anySubscribersWithSkeletonActive = [...subscribers.values()].some(subscriber => (
			subscriber._skeletonSetExplicitly || subscriber._anySubscribersWithSkeletonActive
		));

		this.setSkeletonActive(this._skeletonSetExplicitly || this._anySubscribersWithSkeletonActive || this._skeletonSetByParent);

		subscribers.forEach(subscriber => {
			subscriber.setSkeletonActive(this._skeletonActive);
			subscriber.setSkeletonSetByParent(this._skeletonActive && !subscriber._skeletonSetExplicitly);
		});

		this._parentSkeleton?.registry?.onSubscriberChange();
	}
```

`_skeletonSetExplicitly` is probably overkill, but I always forget what `_skeleton` is. I do think updating `_setByParent` to include skeleton improves the readability a bit, even if it's skeleton overload